### PR TITLE
[16.0][MIG] account_invoice_refund_reason

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
@ForgeFlow

### CODE REVIEW

File: account_invoice_refund_reason/tests/test_account_invoice_refund_reason.py
Comment (BT-anieto) [inline account_invoice_refund_reason/tests/test_account_invoice_refund_reason.py]: Savepoint case is deprecated. Use transaction case instead.

File: account_invoice_refund_reason/wizard/account_move_reversal.py
Comment (BT-anieto) [inline account_invoice_refund_reason/wizard/account_move_reversal.py]: This is wrong. it should be write({"reason_id": self.reason_id **.id** })

File: account_invoice_refund_reason/wizard/account_move_reversal.py
Comment (BT-anieto) [inline account_invoice_refund_reason/wizard/account_move_reversal.py:15]: I guess we should use precompute logic

**Review by BT-anieto — CHANGES_REQUESTED** [review]
Hi @ChrisOForgeFlow Thans for your PR. Could you check my comments, please?

File: account_invoice_refund_reason/wizard/account_move_reversal.py
Comment (BT-anieto) [reply account_invoice_refund_reason/wizard/account_move_reversal.py:15]: Hello @max3903 Am i right? (I'm mentioning you because you appear as maintainer)

File: account_invoice_refund_reason/wizard/account_move_reversal.py
Comment (ChrisOForgeFlow) [reply account_invoice_refund_reason/wizard/account_move_reversal.py]: Done!

File: account_invoice_refund_reason/wizard/account_move_reversal.py
Comment (ChrisOForgeFlow) [reply account_invoice_refund_reason/wizard/account_move_reversal.py:15]: I think this can be handle on another PR to add this feature?

**Comment by BT-anieto** [comment]
Hello @ChrisOForgeFlow ! Thanks for fixing my comments :). There is just two small things that you should do before merge.

1) You cannot have a [FIX] commit. The comments should be ammended to the [MIG] commit and forced pushed to this branch.
2) You need to squash the historic of commits as stated in https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by BT-anieto** [comment]
Hello @ChrisOForgeFlow 

Could you fix my comments? If you don't have time, please tell me and I will close this PR and create another one with the fixes.

**Comment by BT-nfrancisco** [comment]
@ChrisOForgeFlow Hello. Will you take another look at the requested changes of this PR? Thank you a lot.

**Comment by ChrisOForgeFlow** [comment]
> @ChrisOForgeFlow Hello. Will you take another look at the requested changes of this PR? Thank you a lot.

Sorry for delay, please check again

**Review by BT-anieto — COMMENTED** [review]
Hello @ChrisOForgeFlow ! Thank you very much. The squash is not properly done. THe bot commits should e squashed (The ones with [UPD] comming brom odoo bot). Be sure to follow the guide https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate

The rest LGTM!

**Comment by ChrisOForgeFlow** [comment]
> Hello @ChrisOForgeFlow ! Thank you very much. The squash is not properly done. THe bot commits should e squashed (The ones with [UPD] comming brom odoo bot). Be sure to follow the guide https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate
> 
> The rest LGTM!

I'm not sure what commit you refer, can you give me the number of commit should be replaced or squashed?

**Comment by BT-anieto** [comment]
@ChrisOForgeFlow Here I go! If you have more questions, just tell me. As a suggestion, just fork as well the 16.0 branch to your repo, so that you can do git rebase -i origin/16.0. Once forked, I could also do the rebase and show you the final result.

squash1
squash2
squash3

**Comment by BT-anieto** [comment]
@ChrisOForgeFlow Any update here?

**Comment by ChrisOForgeFlow** [comment]
> @ChrisOForgeFlow Any update here?

Now is cleaned, thank you, sorry for delay it's my first clean of those bot's PR

**Comment by BT-anieto** [comment]
Hi @ChrisOForgeFlow Thank you. it is much better but still missing 3 to squash: 
image

**Comment by BT-anieto** [comment]
As well coverage to be fixed

**Comment by pedrobaeza** [comment]
Continued respecting attribution in #1612, so closing.

**Pull request closed without merging**
